packages/core/upload/admin/src/pages/App/components/Header.js

```diff
@@ -87,7 +86,15 @@ Header.defaultProps = {
 Header.propTypes = {
   assetCount: PropTypes.number.isRequired,
   canCreate: PropTypes.bool.isRequired,
-  folder: PropTypes.shape(FolderDefinition),
+  folder: PropTypes.shape({
```

Comment:
Mh, why is that? We are passing an instance of `FolderDefinition` into the component and I'd prefer to keep it like that ...

Reply:
I changed it for the Header because we had lots of React props error coming from it, until I realized we were passing folder from `useFolder` hook and not `useFolders`. I was surprised too but we don't receive as much data about the folder - so different prop definition. Am I missing something? 🤔

Reply:
Mh, if that is the case, what do you think about introducing a `FolderParentDefinition`, which we can re-use in the proptypes?

Reply:
sure sounds good!

Reply:
added [here](https://github.com/strapi/strapi/pull/13610/commits/8b54f5c74f8bc4e43091884ffddd809bf27d1df1)